docs/architecture/raft.rst

```diff
@@ -33,6 +33,9 @@ for information on how to handle failures.
 Note that when you have a two-DC cluster with the same number of nodes in each DC, the cluster will lose the quorum if one
 of the DCs is down.
 **We recommend configuring three DCs per cluster to ensure that the cluster remains available and operational when one DC is down.**
+In the case of two-DC cluster with the same number of nodes in each DC, it is sufficient for the third DC to contain only one node.
+That node can be configured with ``join_ring=false`` and run on a weaker machine.
+See :doc:`Configuration Parameters </reference/configuration-parameters/>` for details about the ``join_ring`` option.
```

Comment:
I sugget, maybe in a followup PR, to excpilitly use term **Arbiter**, and add a page to explain what it is, and how to use it.

Examples for a Doc pages on Arbiter (or similar):
https://www.mongodb.com/docs/manual/core/replica-set-arbiter/
https://dev.mysql.com/doc/mysql-cluster-excerpt/8.0/en/mysql-cluster-ndbd-definition.html#ndbparam-ndbd-arbitration
https://discuss.aerospike.com/t/how-to-configure-a-tie-breaker-node/8908

Reply:
In the new version of the PR, there is no such thing as Arbiter DC. Here is the explanation why: https://github.com/scylladb/scylladb/pull/19684#issuecomment-2288172619.

Now, a user basically needs to add a normal DC. The only difference is that all nodes should be configured with `join_ring=false`. That's it. But now that I think about, maybe it's still good to call such a DC an Arbiter in the documentation and add a new page for it. Even though such DC is not special in any way internally, it has a special purpose.

Also, we could explain other use cases than the two DC setup with an equal number of nodes. In some cases, a user may need to add more that one zero-token node or/and change the voter set (#18793). Also, adding more than one zero-token node is an option to be safe in more extreme disaster scenarios (like 1 DC + 1 node death). But AFAIK, most users have the two DC setup with an equal number of nodes, and we don't want to complicate the documentation for some unreal cases.

Another thing is the documentation of zero-token nodes. A new page for them could be a good idea. There is much more to them than what is explained in the description of the `join_ring` option. I'll think about it.